examples/00-load/read-dolfin.py

```diff
@@ -0,0 +1,32 @@
+"""
+Read FEniCS/Dolfin Meshes
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+PyVista leverages `meshio`_ to read many mesh formats not natively supported
+by VTK including the `FEniCS/Dolfin`_ XML format.
+
+.. _meshio: https://github.com/nschloe/meshio
+.. _FEniCS/Dolfin: https://fenicsproject.org
+"""
+import pyvista as pv
+from pyvista import examples
+
+###############################################################################
+# Let's download an example FEniCS/Dolfin mesh from our example data
+# repository. This will download an XML Dolfin mesh and save it to PyVista's
+# data directory.
+saved_file, _ = examples.downloads._download_file("dolfin_fine.xml")
```

Comment:
you should avoid exposing private functions in the public documentation.

Reply:
Yeah, I agree... I did this to show that we are simply downloading a file and can pass that filename to the `pyvista.read()` method. We could use the `exampled.download_dolfin()` method, but it returns the loaded mesh without showing that it is as simple as passing the file name to `pyvista.read()`

We've been meaning to rewrite the whole downloads API for #210 so maybe we can make these utilities public when we rewrite it